Storage/SQL/SQLiteBookmarksModel.swift

```diff
@@ -192,7 +192,7 @@ open class SQLiteBookmarksModelFactory: BookmarksModelFactory {
             // The buffer never includes the Places root, so we look one level deeper.
             // Because this is a special-case overlay, we include Mobile Bookmarks here --
             // that'll show bookmarks from other mobile devices.
```

Comment:
Probably still worth mentioning that we don't include Mobile Bookmarks, because we're inverting the tree to put them at the top. Folks won't necessarily read the whole pile of code…